src/components/ChatContent/Messages/BlockMenu/ReportDialog.vue

```diff
@@ -0,0 +1,300 @@
+<template>
+  <modal name="reportModal" classes="report-dialog" :maxWidth="350" height="auto" :adaptive="true">
+    <h3 style="text-align: center; margin: 0px;">Ignore/Report User</h3>
```

Comment:
Why? It isn't a lot of css and it isn't repeating. Shouldn't it be here, so that it could be understood at a glance?
I don't really know much about css standards though, so could you explain your thought process about this?

Reply:
Separation of concerns, basically. The style has no bearing on the structure of the page. Makes it easier to read the HTML itself without having to look at the style information as well. It's not a huge deal if it isn't a ton of CSS, but I think it's good practice, especially since Vue has scoped styles. To be honest I tend to go back and forth on what I prefer.

Reply:
I think I disagree with this;
The way I see it classes should provide you with the general information about the style.
So, for example, the green-button class tells you that this div would work like the buttons you see on the site. `class="title"` on the other hand gives you no useful information about the style of that element, and so you are forced to scroll to the css for it.
Specifically with the margin and the display properties, it really isn't obvious what their values are from the class, and I personally think that their value does matter when thinking about the html structure. So since there isn't anything else in the style, I don't really think that they should be separated from the html and replaced by a class.

Reply:
Yeah that seems fair.